inc/saiport.h

```diff
@@ -221,6 +221,19 @@ typedef enum _sai_port_fec_mode_extended_t
     SAI_PORT_FEC_MODE_EXTENDED_FC,
 } sai_port_fec_mode_extended_t;
 
+/**
+ * @brief Attribute data for #SAI_PORT_ATTR_DATAPATH_ENABLE
+ */
+typedef enum _sai_port_datapath_enable_t
+{
+    /** PASS through data path */
+    SAI_PORT_DATAPATH_ENABLE_PASS_THROUGH_MODE,
+
+    /** PCS IEEE data path */
+    SAI_PORT_DATAPATH_ENABLE_PCS_IEEE_MODE,
```

Comment:
```suggestion
    SAI_PORT_DATAPATH_PCS_IEEE_MODE,
```

Reply:
Meta checkes fail if we change the enum names, it should align with attribute name.